### PR TITLE
Add phone number to feedback form

### PR DESCRIPTION
**Pull request by Netsensia**
@brettminnie Please could you cast a quick eye over this and then merge if ok to master - it just adds a phone number to the feedback form and then adds this field to the email template when sending to OPG.

### CODE REVIEW

File: module/Application/src/Application/Form/General/FeedbackForm.php [hunk 1]
```diff
@@ -57,6 +57,11 @@ public function __construct( $formName = 'send-feedback' ){
             'type' => 'Email',
         ));
         
+        $this->add(array(
+            'name' => 'phone',
```
Comment (brettminnie) [inline module/Application/src/Application/Form/General/FeedbackForm.php]: generic pedantry here, if you are splitting declarations by line, split all the things, to make it conformant with PSR-2 ie:

```php
$this->add(
    array(
        'name' => 'phone',
        'type'   => 'Text',
    )
);  
```

File: module/Application/view/application/feedback/index.twig [hunk 2]
```diff
@@ -72,7 +80,7 @@
     </fieldset>
 
     <div class="text">
-        <p>We are always working to improve this service. If you are happy to be contacted to give further feedback, please leave your email address.</p>
+        <p>We are always working to improve this service. If you are happy to be contacted to give further feedback, please leave your email address and/or phone number.</p>
```
Comment (brettminnie) [inline module/Application/view/application/feedback/index.twig]: Should `give further feedback` not be more correct as `provide further feedback`

Comment (Netsensia) [inline module/Application/view/application/feedback/index.twig]: Thanks. I didn't change this text as it was already there - could be worth asking OPG if they want to reword it.

File: module/Application/src/Application/Form/General/FeedbackForm.php (hunk 1 above)
Comment (Netsensia) [inline module/Application/src/Application/Form/General/FeedbackForm.php]: You're right, but in this case this keeps the new statement consistent with the rest of the code in that file.

File: module/Application/view/email/feedback.twig [hunk 3]
```diff
@@ -33,6 +33,15 @@
 </p>
 
 <p style="font-size: 19px; line-height: 1.315; margin: 0 0 20px 0; color: #0b0c0c">
+    <strong>Phone</strong><br>
+    {% if  phone|length > 0 %}
+        {{ phone }}
+    {% else %}
+        No phone number given
```
Comment (Netsensia) [inline module/Application/view/email/feedback.twig]: This was to keep it consistent with the text used for the email field.

File: module/Application/view/application/feedback/index.twig (hunk 2 above)
Comment (brettminnie) [inline module/Application/view/application/feedback/index.twig]: No problem, just reads better, maybe raise it as a query.

**Pull request merged**